src/terminal/scroll/init.lua

```diff
@@ -0,0 +1,84 @@
+--- Terminal Scroll Module.
+-- Provides utilities for handling scroll regions in terminals.
+-- @module scroll
+local M = {}
+M.output = require("terminal.output")
```

Comment:
this is not correct; `terminal.output` is no duplicated as `terminal.scroll.output`.

Just make it a local `output`, or require `terminal` instead and change `M.output(...)` to `terminal.output(...)`